### PR TITLE
(DOC-4056) Add details to import workflow for certs and CRLs

### PR DESCRIPTION
**Pull request by clairecadman**

### CODE REVIEW

**Comment by clairecadman** [comment]
Wait to merge - Paul will also review. Thanks!

File: documentation/intermediate_ca.md [hunk 1]
```diff
@@ -16,19 +16,49 @@ If you have an external certificate authority, you can create a cert chain from
...
+-----END X509 CRL-----
+-----BEGIN X509 CRL-----
+<Org’s intermediate CA CRL>
+-----END X509 CRL-----
+-----BEGIN X509 CRL-----
+<Org’s root CA CRL>
+-----END X509 CRL-----
+```
+
+8. Use the `puppet server can import` command to trigger the rest of the CA setup:
```
Comment (justinstoller) [inline documentation/intermediate_ca.md]: `puppet server can import` is wrong.

**Comment by justinstoller** [comment]
looks good to be save for the one typo I mentioned.

File: documentation/intermediate_ca.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ Puppet Server needs to present the full certificate chain to clients so the clie
 -----END X509 CRL-----
 ```
 
-8. Use the `puppet server can import` command to trigger the rest of the CA setup:
+8. Use the `puppet server import` command to trigger the rest of the CA setup:
```
Comment (justinstoller) [inline documentation/intermediate_ca.md]: sorry, I think that if this is in backticks it usually denotes the command, which is `puppetserver ca import` but if you want to refer to "the puppet server import command to.." it doesn't need to be wrapped in backticks.

Comment (clairecadman) [reply documentation/intermediate_ca.md]: Updated, thanks!

File: documentation/intermediate_ca.md [hunk 3]
```diff
@@ -16,19 +16,49 @@ If you have an external certificate authority, you can create a cert chain from
...
+-----END X509 CRL-----
+-----BEGIN X509 CRL-----
+<Org’s intermediate CA CRL>
+-----END X509 CRL-----
+-----BEGIN X509 CRL-----
+<Org’s root CA CRL>
+-----END X509 CRL-----
+```
+
+8. Use the `puppet server ca import` command to trigger the rest of the CA setup:
```
Comment (Magisus) [inline documentation/intermediate_ca.md]: Nit: there shouldn't be a space between `puppet` and `server`.

**Pull request merged**